inlong-sort/sort-connectors/mysql-cdc/src/main/java/org/apache/inlong/sort/cdc/mysql/source/split/MySqlSplitSerializer.java

```diff
@@ -296,10 +329,12 @@ public MySqlSplit deserializeSplit(int version, byte[] serialized) throws IOExce
             Map<TableId, TableChange> tableChangeMap = readTableSchemas(version, in);
             int totalFinishedSplitSize = finishedSplitsInfo.size();
             boolean isSuspended = false;
+            Map<TableId, String> tableDdls = null;
             if (version >= 3) {
                 totalFinishedSplitSize = in.readInt();
                 if (version > 3) {
                     isSuspended = in.readBoolean();
+                    tableDdls = readTableDdls(version, in);
```

Comment:
what if the job restore from the old job and doesn't have ddl field, will it throw EOFException()?

Reply:
Thanks, it has been fixed.